modules/gitops-argo-cd-installation.adoc

```diff
@@ -19,8 +19,8 @@ To manage cluster configurations or deploy applications, you can install and dep
 
 . Click *Create* to configure the following parameters:
 
-.. For *Name*, enter `/`.
+.. By default the *Name* is set as *argocd*. Enter the *Name* of the instance to change the name.
 
-.. Create an external OS Route to access Argo CD server. Click *Server* -> *Route* and, for *Enable Route*, select *true*.
+.. Click *Server* -> *Route* and, for *Enable Route*, select *true*. By default, the server route is enabled. 
 
 . To open the Argo CD web UI, click the route that is created in the project where the Argo CD instance is installed.
```

Comment:
> To open the Argo CD web UI, click the route that is created in the project where the Argo CD instance is installed.

To open the Argo CD web UI, click the route by navigating to **Networking -> Routes -> < argocd instance name >-server** in the project where the Argo CD instance is installed